Lib/test/libregrtest/runtest.py

```diff
@@ -98,6 +98,13 @@ def runtest(ns, test):
 
     output_on_failure = ns.verbose3
 
+    if ns.huntrleaks:
```

Comment:
If you put the warning here, you get one warning per test file. It would be better to put it in Regrtest.runtest().

Reply:
My bad. I did not notice this because I usually only run one file.